analyzer/codechecker_analyzer/buildlog/log_parser.py

```diff
@@ -699,7 +718,7 @@ def f(flag_iterator, details):
     return f
 
 
-def parse_options(compilation_db_entry):
+def parse_options(compilation_db_entry, compiler_info_file=None):
```

Comment:
Shouldn't we set the default value `compiler_info_file=None` on the class staticmethod?
```
    @staticmethod
    def set(details, compiler_info_file):
```